internal/tools/aws/client_test.go

```diff
@@ -0,0 +1,175 @@
+package aws
+
+import (
+	"sync"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/acm"
+	"github.com/sirupsen/logrus"
+
+	testlib "github.com/mattermost/mattermost-cloud/internal/testlib"
+	"github.com/mattermost/mattermost-cloud/model"
+
+	"github.com/stretchr/testify/suite"
+)
+
+// ClientTestSuite supplies tests for aws package Client.
+type ClientTestSuite struct {
+	suite.Suite
+	session *session.Session
+}
+
+func (d *ClientTestSuite) SetupTest() {
+	d.session = session.Must(session.NewSession())
+}
+
+func TestClientSuite(t *testing.T) {
+	suite.Run(t, new(ClientTestSuite))
+}
+
+type Mocks struct {
+	AWS   *Client
+	API   *testlib.AWSMockedAPI
+	LOG   *testlib.MockedFieldLogger
```

Comment:
LOG should be Log to follow Go style, no all-caps variables unless they are acronyms